tests/rptest/redpanda_cloud_tests/high_throughput_test.py

```diff
@@ -170,10 +170,17 @@ class HighThroughputTest(RedpandaTest):
 
     def __init__(self, test_ctx: TestContext, *args, **kwargs):
         self._ctx = test_ctx
-
         # Default set to tier-1-aws is a temporary work around for
         # https://github.com/redpanda-data/cloudv2/issues/7903
-        cloud_tier_str = test_ctx.globals.get("cloud_tier", "tier-1-aws")
+        if RedpandaServiceCloud.GLOBAL_CLOUD_CLUSTER_CONFIG in self._ctx.globals:
+            # Load needed config values from cloud section
+            # of globals prior to actual cluster creation
+            cloud_tier_str = self._ctx.globals[
```

Comment:
both settings of `cloud_tier_str` are now local conditional branch variables that won't exist when set on line 184 (unless Python does something wierd?)

Reply:
Python3 treats it like globals inside an conditional scope and cycles

Reply:
true, for me its a style thing, i guess we could just do
```
cloud_tier_str = "tier-1-aws"
if RedpandaServiceCloud.GLOBAL_CLOUD_CLUSTER_CONFIG in self._ctx.globals:
   cloud_tier_str = ......
# Omit the else clause
```